crates/ruma-state-res/src/room_version.rs

```diff
@@ -98,12 +98,7 @@ impl RoomVersion {
         ..Self::V5
     };
 
-    pub const V7: Self = Self {
-        // FIXME: once room version 7 is stabilized move this to version 8
-        disposition: RoomDisposition::Unstable,
-        allow_knocking: true,
-        ..Self::V6
-    };
+    pub const V7: Self = Self { allow_knocking: true, ..Self::V6 };
```

Comment:
Is 8 still unstable, not that anything uses it but should we move the `disposition: RoomDisposition::Unsable` to the next room version?

Reply:
Stable: https://spec.matrix.org/v1.2/rooms/#complete-list-of-room-versions